source/SIL.AppBuilder.Portal.Frontend/src/data/with-stubbed-dev-data.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { withData, WithDataProps } from 'react-orbitjs';
+import { compose } from 'recompose';
+
+// Don't have API data yet?
+// Stub it with this!
+export function withStubbedDevData(type: string, forcedId: string, attributes: any) {
```

Comment:
I needed this yesterday... I created data in the componentWillMount

Reply:
Yeah, I was almost doing that, too. Decided to come up with something more flexible.